corona-warn-app/cwa-app-android
# Request DCC Screen (EXPOSUREAPP-7549, 7553)

Connects Request DCC screen to submission flow

### Testing 
- Check figma flow when there a test of same type exists and not 
- Register RAT(with dcc and without) and PCR tests and check the correctness of the flow
- Check other paths such `X` dialog
